src/tests/responseTypes/responseTypes.ts

```diff
@@ -0,0 +1,8 @@
+
+export default interface responseType<Type> {
```

Comment:
I wasn't quite finding what I needed in https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types, just getting lost in various `request` libs that are either deprecated or different, so I wrote a quick generic.